pkg/validation/options.go

```diff
@@ -107,28 +105,6 @@ func Validate(o *options.Options) error {
 	return nil
 }
 
-func parseSignatureKey(o *options.Options, msgs []string) []string {
-	if o.SignatureKey == "" {
-		return msgs
-	}
-
-	logger.Print("WARNING: `--signature-key` is deprecated. It will be removed in a future release")
-
-	components := strings.Split(o.SignatureKey, ":")
```

Comment:
@JoelSpeed I took the liberty to remove `signature-key` as well. Same question as for the GCPHealthCheck flag.